ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4611,4 +4611,17 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+
+#AHU canonical#  
+AHU_DFSS_DFVSC_FDPM_CHWDC_SRC_DFRMM:
```

Comment:
Youve defined this without important AHU characteristics, particularly outside air control. Is this an AHU or a FCU?

Reply:
It is a ceiling suspended AHU and do not contain any outside air control associated to IT. This is particularly serving the IDF rooms.

Reply:
then it is a FCU. please adjust.